datapackage.json

```
@@ -385,13 +385,17 @@
             "name": "VoterType",
             "type": "string",
             "title": "Voter Type",
-            "description": "Specifies whether the voter is domestic or overseas military (or, if the domestic or overseas designation is unknown, simply whether the voter is military), overseas citizen, or other.",
+            "description": "Specifies whether the voter falls into one of the following designations.",
```

I know I was influential with the creation of the existing line, so I feel bad that I'm thinking of a new approach. How about this?

```suggestion
            "description": "Specifies whether the voter is domestic or overseas military (or simply "military" if the designation is unknown), a spouse or dependent of a member of the military, or other if the type of voter falls outside the previous designations.",
```

---

Looks good, except we are now missing `overseas-citizen` voters in the big 'ol list.